doc/proposals/restoration.md

```diff
@@ -0,0 +1,44 @@
+# Etcd data restoration
```

Comment:
Add reference to this doc in [this section](https://github.com/gardener/etcd-backup-restore/blob/master/doc/usage/getting_started.md#etcd-data-directory-initialization) as well.

Reply:
Done